write_buffer/src/kafka/record_aggregator.rs

```diff
@@ -131,7 +132,14 @@ impl Aggregator for RecordAggregator {
             //
             // Map to the original input op this fn was called with, discarding
             // the encoded Record.
-            TryPush::NoCapacity(_) => TryPush::NoCapacity(op),
+            TryPush::NoCapacity(_) => {
+                // Log a warning if this occurs - this allows an operator to
+                // increase the maximum Kafka message size, or lower the linger
+                // time to minimise latency while still producing large enough
+                // batches for it to be worth while.
+                warn!("aggregated batch reached maximum capacity");
```

Comment:
```suggestion
                warn!("aggregated batch reached maximum capacity. Try increasing maximum Kafka message size or lower the linger time");
```

I think it might be a good idea to include the suggested actions in the message itself so a user doesn't have to know to look in the source code to find them

Reply:
Some settings are in Kafka itself, some are hard-coded, some are exposed via envs (that change) - I fear being more helpful might be less helpful :(